gwcs/utils.py

```diff
@@ -7,15 +7,11 @@
 import functools
 import numpy as np
 from astropy.modeling import models as astmodels
-from astropy.modeling.models import Mapping
-from astropy.modeling import core, projections
+from astropy.modeling import core, projections, is_separable
```

Comment:
Missing import is a real bug found by my `flake8` checker. If you want, I can put this in as separate PR. Though none of your existing tests caught it, so probably not important?